Initial Certification

ekersey opened the pull request:
Request to certify Get Products, Get ProductPricing, GetProductQuantity, Put Customer, and Get Fulfillment

ekersey commented on package.json:
```diff
@@ -3,7 +3,9 @@
-    "request": "2.83.0"
+    "lodash.get": "^4.4.2",
+    "request": "2.83.0",
+    "request-promise-native": "1.0.5"
```
change to require "request-promise"

tgilkerson commented on package.json:
```diff
@@ -14,6 +16,9 @@
     "mocha": "^3.1.2"
   },
+  "engines": {
+    "node": ">=7.6.0"
```
This should be added to the template package.json file

ekersey commented:
I made all of the discussed changes and the PR has been updated.

Removed unused channel action from channel-settings.
Added all known can/do flags to channel-settings.
Replaced lodash.get with jsonata for extracting business references.
Replaced request-promise-native with request-promise.

Updated GetProductQuantity to reflect recent changes to the api that allow filtering on date quantity was modified.

Updated tests for GetProductPricing to mock the expected response once timestamps are added to prices.  This allows the tests to pass, while if used live it will always return a 204 (no pricing updates found) until the timestamps are implemented by iQmetrix.
